### ISSUE TITLE
[REQ] Add to Homebrew-Cask

### ISSUE THREAD

**Issue by ambrosso** [glushchenko/fsnotes#200]
Add app to Homebrew-Cask for convenience

**Comment by jeff-h** [comment]
It's on the App Store already which is pretty convenient, I think.

**Comment by jeff-h** [comment]
@iskrisis || @ambrosso — either of you have experience actually doing this? I've only ever used brew as an end-user. Would we need a cask, a formula, a tap?

**Comment by glushchenko** [comment]
https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md

**Comment by iskrisis** [comment]
Yes this can be a problem. I did it while ago but i have no idea about Ruby so it was a pain and i did it wrong. Also i am not sure if you don't have to have prebuild binary - i am not sure if you can make brew cask that also builds the app.

**Comment by jeff-h** [comment]
Thanks! As far as I can see from those docs, the cask file for FSNotes would reference a pre-built binary, and would thus need to be updated every time a new version is released. @glushchenko you up for that? It'd be relatively trivial, but sounds a bit tedious to me... shame it can't automatically fetch the latest version from https://github.com/glushchenko/fsnotes/releases without having to update the cask file all the time.

I personally do like the notion of brew distribution of FSNotes; although I doubt I'd use it myself, I think it suits the whole "power-user's note taking app" thing FSNotes has going on :) However, I'm uncertain how many people would actually use homebrew to get FSNotes, and it also deprives whatever thanks @glushchenko gains from the MAS sales.

Perhaps we should run a thumbs-up vote on the original comment above, to ascertain if there's widespread interest, or if it's too niche? (apologies to @ambrosso and @iskrisis!)

**Comment by glushchenko** [comment]
I vote for free distribution in brew cask, it is not a problem.

**Comment by gingerbeardman** [comment]
@iskrisis latest (free) version is always here: https://github.com/glushchenko/fsnotes/releases

but yes cask needs updating with every version. maybe a copy of the cask file can be hosted in the repository and updated with the build process?

**Comment by iskrisis** [comment]
I went through the guide and their tests so it is valid for submitting.

https://gist.github.com/iskrisis/7f0295c9f72e7f912af72381763cc33b

Not sure about updates - the things that must be bumped are version, sha256 and url
but from this
https://github.com/Homebrew/homebrew-cask/blob/master/CONTRIBUTING.md#updating-a-cask
it seems that they have script that anyone can just run and does all of the changes including push to their repo. So i asume after submitting then it can be completely separate.

I guess it is a matter of someone checking/testing this gist in brew and publishing the file to homebrew repo.

**Comment by iskrisis** [comment]
Ok formula got successfully merged to homebrew cask core.
https://github.com/Homebrew/homebrew-cask/pull/49247
I will try to keep eye on it from time to time to update it.
There is this easy solution for updating
https://github.com/Homebrew/homebrew-cask/blob/master/CONTRIBUTING.md#updating-a-cask
if somebody notices that.

**Comment by iskrisis** [comment]
I think this can be closed.
